dashboard/static/css/qa_updates_style.css

```diff
@@ -0,0 +1,68 @@
+/* Styling for QA Updates */
+
+.page-title {
+    float:left;
+    font-weight: bold;
+    font-size: 30px;
+    color: black;
+    padding-left: 20px;
+    padding-bottom: 10px;
+  }
+
+.scroll-box {
+    height: 650px;
+    width: 97%;
+    border: 1px solid #ccc;
+    overflow: auto;
+    margin-left: 20px;
+    margin-right: 20px;
+    padding-left: 20px;
+    background-color: #eee;
```

Comment:
looking at the image, maybe a padding or margin at the bottom here will make it a bit better?